tedana/decomposition/eigendecomp.py

```diff
@@ -202,11 +202,9 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
         vTmix = comp_ts.T
         vTmixN = ((vTmix.T - vTmix.T.mean(0)) / vTmix.T.std(0)).T
         LGR.info('Making initial component selection guess from PCA results')
-        _, ct_df, betasv, v_T = model.fitmodels_direct(catd, comp_ts.T, eimum,
-                                                       t2s, t2sG,
-                                                       tes, combmode, ref_img,
-                                                       mmixN=vTmixN,
-                                                       full_sel=False)
+        _, ct_df, betasv, v_T = model.fitmodels_direct(
```

Comment:
I find this convention (where all of the arguments go on a newline) really hard to read -- none of them will fit on the first line, though ?

Reply:
The two ways we've done it in the past are either:
```
_, ct_df, betasv, v_T = model.fitmodels_direct(
    catd, comp_ts.T, eimum, t2s, t2sG, tes, combmode,
    ref_img, mmixN=vTmixN, full_sel=False, label='pca')
```
or
```
_, ct_df, betasv, v_T = model.fitmodels_direct(catd, comp_ts.T, eimum,
                                               t2s, t2sG, tes,
                                               combmode, ref_img,
                                               mmixN=vTmixN,
                                               full_sel=False,
                                               label='pca')
```

I prefer the former because the latter looks kind of bunched up and generally takes up more lines. I hadn't thought about readability though. Do you prefer the second one?

Reply:
I do, personally, but I totally see your point ! Maybe there's a compromise. Could we indent the first one a little more, to show that they're arguments ?
```
_, ct_df, betasv, v_T = model.fitmodels_direct(
                catd, comp_ts.T, eimum, t2s, t2sG, tes, combmode,
                ref_img, mmixN=vTmixN, full_sel=False, label='pca')
```

I don't think that will trigger pylint...

Reply:
It looks like that's safe. I've changed the fitmodels_direct calls.